src/django/api/templates/mail/boundary_needs_revisions_body.txt

```diff
@@ -0,0 +1,6 @@
+Submission for needs revisions - {{city}} {{pwsid}}
+
+The boundary you submitted has been marked-up with revisions.
```

Comment:
We should also include the note that the Validator types as they request changes, since we imply that will be emailed to the Contributor(s):

![image](https://user-images.githubusercontent.com/1430060/201954601-88cdcbc6-af66-4e66-aefe-4816dbd2c3e3.png)

Reply:
[d1b65cf](https://github.com/azavea/iow-boundary-tool/pull/203/commits/d1b65cf33c6fb2cc4593dc2ed621ebab67c2359a)

Reply:
Let's visually indent the notes, by adding four spaces before, to indicate a quote:

```
    {{ review_notes }}
```

this should be good enough for now.